packages/EasyActivity/src/Bridge/Symfony/Serializers/SymfonyActivitySubjectDataSerializer.php

```diff
@@ -71,8 +71,10 @@ public function serialize(array $data, ActivitySubjectInterface $subject): ?stri
             if (\is_object($value)) {
                 $objectClass = \get_class($value);
 
-                $context[AbstractNormalizer::ATTRIBUTES][$key] = $nestedObjectAllowedProperties[$objectClass]
-                    ?? $allowedProperties[$key] ?? ['id'];
+                if (\is_subclass_of($objectClass, \DateTimeInterface::class) === false) {
```

Comment:
It seems we are trying to avoid adding `'id'` to the context because it may not be used. But is it a real problem?
There are a few possible cases:
* `DateTimeNormalizer` is added to `Serializer`. In this case, `AbstractNormalizer::ATTRIBUTES` will be ignored.
* `ObjectNormalizer` is added to `Serializer`. In this case, `id` in `AbstractNormalizer::ATTRIBUTES` will be ignored and an empty object will be returned.
* Normalizers are not defined. An exception will occur.

So, I don't see any issues with adding `id` to the context. And perhaps we don't have to complicate the code.

If we anyway want to keep the context clean, what about relying on `$id` property? For example, we can check it using `symfony/property-access`.

Reply:
The initial reason for this fix was an incorrect context. I'm talking about "incorrect" because according to this context we require the `id` property on objects which may not have this `id` at all. It's not a critical problem, you are right.
But it seemed a little strange to me when debugging.

So my point is that we shouldn't "require" non-existent attributes, even if it doesn't cause issues. But also I believe you'll make a more correct decision about this since you know this feature better than us.

I believe we shouldn't use `is_subclass_of` here and should use `symfony/property-access` as you said.

Reply:
When using `symfony/property-access` it couse to many edge cases, as `$value` may be `ArrayCollection` ...

I prefer to use `is_subclass_of` or just close this PR for now.